src/ui/Startup.cs

```diff
@@ -123,6 +122,12 @@ public void ConfigureServices(IServiceCollection services)
                                 // between cookie and access token is preserved
                                 x.ShouldRenew = true;
                             }
+                            else 
+                            {
+                                // could not refresh - log the user out
+                                _logger.LogWarning("Token refresh failed with message: " + response.ErrorDescription);
```

Comment:
Can we log the subject id as well

Reply:
I don't think the subjectid is in scope at that point - if I understand right, we'd have to query the API to figure this out which I don't want to do

Reply:
Its in scope
```
var identity = (ClaimsIdentity) x.Principal.Identity;
                            var accessTokenClaim = identity.FindFirst("access_token");	                            var accessTokenClaim = identity.FindFirst("access_token");
```